documentation/devguide.rst

```diff
@@ -19,10 +19,8 @@ main Python documentation, except for some small differences.  The source
 lives in a `separate repository`_ and bug reports should be submitted to the
 `devguide GitHub tracker`_.
 
-Our devguide workflow uses continuous integration and deployment so changes to
-the devguide are normally published when the pull request is merged. Changes
-to CPython documentation follow the workflow of a CPython release and are
-published in the release.
+Our devguide follows a continuous integration and deployment workflow, with changes published when pull requests are merged.
+CPython documentation, on the other hand, is updated daily and aligns with the CPython release workflow, with changes being included in each release.
```

Comment:
The "aligns with the CPython release workflow" wording is pretty vague and if I didn't already know how it works, it wouldn't tell me much. The docs are versioned, in that there is a separate documentation for each CPython version, and the online documentation is updated continually (but with a delay as Hugo discussed). Each CPython release also comes with documentation, which may be used by redistributors.

Reply:
@hugovk @JelleZijlstra I made some changes aiming to clarify the delay and versioning at this [commit](https://github.com/python/devguide/pull/1395/commits/34dc1ce25fd13e3e0d15d8f5dd47f919e3f2bacf).
I look forward to suggestions for improvement, thank you for your support.